src/hooks/auditorium/useAuditoriumGrid.tsx

```diff
@@ -1,32 +1,32 @@
 import React, { useMemo } from "react";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 import { GridPosition } from "types/grid";
 
 import { UserProfilePicture } from "components/molecules/UserProfilePicture";
 
 import { GetUserByPosition } from "../useGetUserByPosition";
 
+import styles from "./seats.module.scss";
+
 export interface UseAuditoriumGridProps {
   isUserAudioMuted: boolean;
-  rows: number;
-  columns: number;
   getUserBySeat: GetUserByPosition;
-  checkIfSeat: (gridData: GridPosition) => boolean;
   takeSeat: (gridData: GridPosition) => Promise<void> | undefined;
 }
 
 export const useAuditoriumGrid = ({
   isUserAudioMuted,
-  rows,
-  columns,
-  checkIfSeat,
   getUserBySeat,
   takeSeat,
-}: UseAuditoriumGridProps) =>
-  useMemo(
+}: UseAuditoriumGridProps) => {
+  const rows = 9;
+  const columns = 24;
```

Comment:
There used to be AUDITORIUM_CONSTANTS_NAMED_LIKE_THIS in `settings.ts` now probably still in that `settings/index.ts` file. These numbers look like they should be placed there (hopefully into something like `auditoriumSettings.ts`